diff --git a/node_modules/react-native-input-credit-card/src/CreditCardInput.js b/node_modules/react-native-input-credit-card/src/CreditCardInput.js
index 49971cd..27670f4 100644
--- a/node_modules/react-native-input-credit-card/src/CreditCardInput.js
+++ b/node_modules/react-native-input-credit-card/src/CreditCardInput.js
@@ -21,26 +21,39 @@ const s = StyleSheet.create({
     alignItems: "center",
   },
   form: {
-    marginHorizontal: 20,
-    marginVertical: 20,
+    // marginHorizontal: 20,
+    // marginVertical: 20,
+    width: 300 - 5,
+    
+  },
+  formContent:{
+
   },
   inputContainer: {
-    marginLeft: 20,
-    marginTop: 20,
-    marginBottom: 20,
+    // marginTop: 20,
+    // marginBottom: 20,
+    marginTop: 10,
+    marginBottom: 10,
   },
   inputLabel: {
     fontWeight: "bold",
+    fontFamily:'workSansMedium',
+    fontSize: 14,
+    textTransform: 'capitalize'
   },
   input: {
-    height: 40,
+    // height: 40,
+    height: 30,
+    padding: 5
   },
 });
 
-const CVC_INPUT_WIDTH = 70;
-const EXPIRY_INPUT_WIDTH = CVC_INPUT_WIDTH;
+// const CVC_INPUT_WIDTH = 70;
+const CVC_INPUT_WIDTH = Dimensions.get("window").width * 0.25;
+const EXPIRY_INPUT_WIDTH = Dimensions.get("window").width * 0.5;
 const CARD_NUMBER_INPUT_WIDTH_OFFSET = 40;
-const CARD_NUMBER_INPUT_WIDTH = Dimensions.get("window").width * 0.5;
+// const CARD_NUMBER_INPUT_WIDTH = Dimensions.get("window").width * 0.5;
+const CARD_NUMBER_INPUT_WIDTH = Dimensions.get("window").width ;
 const NAME_INPUT_WIDTH = CARD_NUMBER_INPUT_WIDTH;
 const PREVIOUS_FIELD_OFFSET = 40;
 const POSTAL_CODE_INPUT_WIDTH = 120;
@@ -80,7 +93,7 @@ export default class CreditCardInput extends Component {
       name: "NAME",
       number: "CARD NUMBER",
       expiry: "EXPIRY",
-      cvc: "CVC/CCV",
+      cvc: "CVC/CVV",
       postalCode: "POSTAL CODE",
     },
     placeholders: {
@@ -92,7 +105,7 @@ export default class CreditCardInput extends Component {
     },
     inputContainerStyle: {
       borderBottomWidth: 1,
-      borderBottomColor: "black",
+      borderBottomColor: "#DADAED",
     },
     validColor: "",
     invalidColor: "red",
@@ -110,15 +123,15 @@ export default class CreditCardInput extends Component {
   _focus = field => {
     if (!field) return;
 
-    const scrollResponder = this.refs.Form.getScrollResponder();
-    const nodeHandle = findNodeHandle(this.refs[field]);
+    // const scrollResponder = this.refs.Form.getScrollResponder();
+    // const nodeHandle = findNodeHandle(this.refs[field]);
 
-    NativeModules.UIManager.measureLayoutRelativeToParent(nodeHandle,
-      e => { throw e; },
-      x => {
-        scrollResponder.scrollTo({ x: Math.max(x - PREVIOUS_FIELD_OFFSET, 0), animated: true });
-        this.refs[field].focus();
-      });
+    // NativeModules.UIManager.measureLayoutRelativeToParent(nodeHandle,
+    //   e => { throw e; },
+    //   x => {
+    //     scrollResponder.scrollTo({ x: Math.max(x - PREVIOUS_FIELD_OFFSET, 0), animated: true });
+    //     this.refs[field].focus();
+    //   });
   }
 
   _inputProps = field => {
@@ -197,8 +210,10 @@ export default class CreditCardInput extends Component {
           scrollEnabled={allowScroll}
           showsHorizontalScrollIndicator={false}
           style={s.form}
+          contentContainerStyle={s.formContent}
           {...scrollViewProps}
         >
+          <View>
           {requiresName && (
             <CCInput
               {...this._inputProps("name")}
@@ -210,24 +225,27 @@ export default class CreditCardInput extends Component {
             keyboardType="numeric"
             containerStyle={[s.inputContainer, inputContainerStyle, { width: CARD_NUMBER_INPUT_WIDTH }]}
           />
+          <View style={{flexDirection:'row'}}>
           <CCInput
             {...this._inputProps("expiry")}
             keyboardType="numeric"
-            containerStyle={[s.inputContainer, inputContainerStyle]}
+            containerStyle={[s.inputContainer, inputContainerStyle, {width: EXPIRY_INPUT_WIDTH}]}
           />
           {requiresCVC && (
             <CCInput
               {...this._inputProps("cvc")}
               keyboardType="numeric"
-              containerStyle={[s.inputContainer, inputContainerStyle, { width: CVC_INPUT_WIDTH }]}
+              containerStyle={[s.inputContainer, inputContainerStyle, { width: CVC_INPUT_WIDTH, marginLeft: 20, textTransform: 'uppercase' }]}
             />
           )}
+          </View>
           {requiresPostalCode && (
             <CCInput
               {...this._inputProps("postalCode")}
               containerStyle={[s.inputContainer, inputContainerStyle, { width: POSTAL_CODE_INPUT_WIDTH }]}
             />
           )}
+          </View>
         </ScrollView>
       </View>
     );
